webapp-django/crashstats/signature/static/signature/js/signature_tab_correlations.js

```diff
@@ -21,19 +22,72 @@ SignatureReport.CorrelationsTab = function (tabName) {
 
 SignatureReport.CorrelationsTab.prototype = SignatureReport.inherit(SignatureReport.Tab.prototype);
 
+SignatureReport.CorrelationsTab.prototype.loadControls = function() {
+    var self = this;
+
+    var defaultProduct = $('#mainbody').data('default-product');
+    var defaultChannel = $('#mainbody').data('default-channel');
```

Comment:
This is not working as expected, for example if you open `/signature/?version=50.0b&signature=std%3A%3Alist%3CT%3E%3A%3Aclear%20|%20CDeviceChild%3CT%3E%3A%3A~CDeviceChild%3CT%3E&date=%3E%3D2016-10-06T11%3A03%3A00.000Z&date=%3C2016-10-13T11%3A03%3A00.000Z#correlations` the default channel is `nightly`, but it should be `beta`.